homeassistant/components/facebox/image_processing.py

```diff
@@ -168,7 +170,8 @@ def setup_platform(hass, config, add_entities, discovery_info=None):
     port = config[CONF_PORT]
     username = config.get(CONF_USERNAME)
     password = config.get(CONF_PASSWORD)
-    url_health = f"http://{ip_address}:{port}/healthz"
+    protocol = config.get(CONF_PROTOCOL, "https")
```

Comment:
Checking it during the configuration validation.

```suggestion
    protocol = config.get(CONF_PROTOCOL)
```